### ISSUE TITLE
I would like to ask if you have encountered any such problems? My S3DISDataLoader file is working fine，But the following mistakes were encountered during training。

### ISSUE THREAD

**Issue by xiaoyangyang123** [ShenZheng2000/PointNorm-for-Point-Cloud-Analysis#5]
(point) D:\YN\Pointnet_Pointnet2_pytorch-master>python train_semseg.py --model pointnet2_sem_seg 
PARAMETER ...
Namespace(batch_size=16, color_contrast=False, color_drop=False, color_normalize=False, decay_rate=0.0001, epoch=32, epsilon=0.1, gpu='0', learning_rate=0.001, log_dir='outputs', lr_de
cay=0.7, model='pointnet2_sem_seg', npoint=4096, optimizer='AdamW', pc_rotation=True, scale_factor=1, scheduler='cos', step_size=10, test_area=5)
start loading training data ...
100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 204/204 [00:13<00:00, 14.85it/s] 
[1.124833  1.1816078 1.        2.2412012 2.340336  2.343587  1.7070498
 2.0335796 1.8852289 3.8252103 1.7948895 2.7857335 1.3452303]
Totally 47623 samples in train set.
start loading test data ...
100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 68/68 [00:05<00:00, 12.27it/s] 
[ 1.1427666  1.2085758  1.        10.0776825  2.5504515  2.0249689
  2.1271675  1.9817276  2.5017133  4.765974   1.4132063  2.9075055
  1.484701 ]
Totally 19191 samples in test set.
The number of training data is: 47623
The number of test data is: 19191
No existing model, starting training from scratch...
**** Epoch 1 (1/32) ****
BN momentum updated to: 0.100000
  0%|                                                                                                                                                         | 0/2976 [00:00<?, ?it/s]O
MP: Error #15: Initializing libiomp5md.dll, but found libiomp5md.dll already initialized.
OMP: Hint This means that multiple copies of the OpenMP runtime have been linked into the program. That is dangerous, since it can degrade performance or cause incorrect results. The b
est thing to do is to ensure that only a single OpenMP runtime is linked into the process, e.g. by avoiding static linking of the OpenMP runtime in any library. As an unsafe, unsupport
ed, undocumented workaround you can set the environment variable KMP_DUPLICATE_LIB_OK=TRUE to allow the program to continue to execute, but that may cause crashes or silently produce i
ncorrect results. For more information, please see http://www.intel.com/software/products/support/.

**Comment by ShenZheng2000** [comment]
According to this [post](https://stackoverflow.com/questions/20554074/sklearn-omp-error-15-initializing-libiomp5md-dll-but-found-mk2iomp5md-dll-a), you can add the following lines for a quick fix. 

```
import os
os.environ["KMP_DUPLICATE_LIB_OK"]="TRUE"
```

**Comment by xiaoyangyang123** [comment]
> 

非常感谢，已成功开始运行

**Comment by xiaoyangyang123** [comment]
作者您好，我想跟你确认一下，当我运行测试代码（python test_semseg.py --log_dir {name_for_your_log_dir} --visual # with visualization）时，最终的这几个结果是否就对应于文章中的结果？如果有错误，请您指正一下。









------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "ShenZheng2000/PointNorm-for-Point-Cloud-Analysis"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年6月6日(星期二) 凌晨1:51
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [ShenZheng2000/PointNorm-for-Point-Cloud-Analysis] I would like to ask if you have encountered any such problems? My S3DISDataLoader file is working fine，But the following mistakes were encountered during training。 (Issue #5)





 
According to this post, you can add the following lines for a quick fix.
 import os os.environ["KMP_DUPLICATE_LIB_OK"]="TRUE"

**Comment by ShenZheng2000** [comment]
Yes. Executing this command will yield the .obj files. You can then view these in 3D using Meshlab. Finally, capture a few screenshots that will allow you to produce 2D results for visual comparisons.
